gcc/config/riscv/riscv.md

```diff
@@ -1840,7 +1840,7 @@
 			   (match_operand:QI 2 "immediate_operand" "I"))
 		(match_operand 3 "immediate_operand" "")))
    (clobber (match_scratch:DI 4 "=&r"))]
-  "TARGET_64BIT && !TARGET_BITMANIP
+  "TARGET_64BIT && !(TARGET_ZBB || TARGET_ZBP)
```

Comment:
Actually expands to addu.w which is ZBB and ZBA.  But optionally could expand to pack with more work.

Reply:
Fixed